.chloggen/otelcol_version.yaml

```diff
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: builder
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Remove builder support to build old version, and the otelcol_version config
+
+# One or more tracking issues or pull requests related to the change
+issues: [11405]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: User should remove this property from their config, to build older versions use older builders.
```

Comment:
We should have an error that specifically tells you this for a few releases instead of "unknown configuration setting"

Reply:
Done